### ISSUE TITLE
Flashing/honking feature not working

### ISSUE THREAD

**Issue by dwapps** [lendy007/homeassistant-skodaconnect#66]
**Describe the bug**
The new flashing/honking integration doesn't seem to work for me (Superb iV)
It's working via the MySkoda app. 

**Debug logs**
```
2021-09-06 21:18:46 WARNING (MainThread) [skodaconnect.vehicle] Failed to flash vehicle - name 'latitude' is not defined
2021-09-06 21:18:46 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [***************] Honk and flash action failed
Traceback (most recent call last):
File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 185, in handle_call_service
await hass.services.async_call(
File "/usr/src/homeassistant/homeassistant/core.py", line 1492, in async_call
task.result()
File "/usr/src/homeassistant/homeassistant/core.py", line 1527, in _execute_service
await handler.job.target(service_call)
File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 213, in handle_service
await self.hass.helpers.service.entity_service_call(
File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 658, in entity_service_call
future.result() # pop exception if have
File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 843, in async_request_call
await coro
File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 695, in _handle_entity_call
await result
File "/config/custom_components/skodaconnect/switch.py", line 57, in async_turn_on
await self.instrument.turn_on()
File "/usr/local/lib/python3.9/site-packages/skodaconnect/dashboard.py", line 417, in turn_on
await self.vehicle.set_honkandflash('flash')
File "/usr/local/lib/python3.9/site-packages/skodaconnect/vehicle.py", line 1026, in set_honkandflash
raise SkodaException('Honk and flash action failed')
skodaconnect.exceptions.SkodaException: Honk and flash action failed
```

**Installation:**
 - Home Assistant variant: Hass OS
 - Python version 3.9.6

**Comment by Farfar** [comment]
Looks like I forgot to change variable names at one location. Fix scheduled for next release.

**Comment by Farfar** [comment]
Should be fixed in latest release, v1.0.55
